spec/features/feedback_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'Feedback', type: :feature do
+  before(:each) { visit feedback_url }
+  
+  it 'Submit feedback' do
+    @feedback = Feedback.new(:name => "Joe Public", :email => "foo@example.com",
+      :comment => "I can't submit a petition for some reason", :petition_link_or_title => 'link')
+
+    fill_in "feedback[email]", :with => @feedback.email
+    fill_in "feedback[petition_link_or_title]", :with => @feedback.petition_link_or_title
+    fill_in "feedback[comment]", :with => @feedback.comment
+
```

Comment:
With regards to the fill_in can we use the relevant label name for example `fill_in 'Comments'` to make this a little more readable where possible.